cancermuts/datasources.py

```diff
@@ -907,96 +907,92 @@ def _get_revel(self, mutation):
 
         revel_score = None
 
-        mutation.metadata['revel_score'] = list()
+        mutation.metadata['revel_score'] = []
 
-        if 'genomic_coordinates' not in mutation.metadata and 'genomic_mutations' not in mutation.metadata:
-            self.log.warning("no genomic coordinates or genomic mutations data for mutation %s; it will be skipped" % mutation)
+        gcs = mutation.metadata.get('genomic_coordinates', [])
+        gms = mutation.metadata.get('genomic_mutations', [])
+
+        if (gcs is None or gcs == [] or gcs == '') and (gms is None or gms == [] or gms == ''):
```

Comment:
why do we check for empty strings? Do we have cases in which this is actually happening? I was wondering because I don't think we should ever have that, if we have that it might be an issue